src/coreclr/inc/corinfo.h

```diff
@@ -646,6 +646,8 @@ enum CorInfoHelpFunc
     CORINFO_HELP_VALIDATE_INDIRECT_CALL,    // CFG: Validate function pointer
     CORINFO_HELP_DISPATCH_INDIRECT_CALL,    // CFG: Validate and dispatch to pointer
 
+    CORINFO_HELP_STATIC_VIRTUAL_AMBIGUOUS_RESOLUTION, // Throw AmbiguousResolutionException for failed static virtual method resolution
+
```

Comment:
Sorry, just noticed this during a conflict resolution -- but this should get a matching change in CorInfoHelpFunc.cs, and I think it should also get a JIT-EE GUID update.

Reply:
If you plan to submit the JIT-EE GUID update, then can you also please take care of #69900 at the same time?

Reply:
In fact, it would be nice to name this `CORINFO_HELP_THROW_AMBIGUOUS_RESOLUTION_EXCEPTION` and move it up together with the other `CORINFO_HELP_THROW_*` helpers at the same time.